Store.js

```diff
@@ -102,6 +112,11 @@ define(["dcl/dcl", "decor/Invalidating"], function (dcl, Invalidating) {
 		 */
 		computeProperties: function (props) {
 			if ("store" in props || "query" in props) {
+				if (!Array.isArray(this.store)) {
+					this._storeAdapter = new DstoreToStoreAdapter({store: this});
+				} else {
+					this._storeAdapter = new ArrayToStoreAdapter({store: this});
+				}
```

Comment:
Hmm, I just noticed that you are still passing in the the delite/Store as a property to the adapters.  As I've said before, the idea is for the adapters to be independent from the delite/Store code.   I thought you were passing in `this.store` not `this`.